### DISCUSSION TITLE
RMK툴로도 왈도 서식으로 출력하게 해줘 뺴애액

### DISCUSSION THREAD

**Discussion by zilrty** [RimWorldKorea/RMK#331] [질문]
그때 csv파일로 왈도서식에 적용하는법 알려준거 봐도 뭔소린지 모르겠음

**Comment by RimWorldKorea** [comment]
가장 최근에 작업한 [파일](https://github.com/RimWorldKorea/RMK/files/13735044/example.xlsx) 참고

<img width="50%" alt="화면 캡처 2023-12-21 130524" src="https://github.com/RimWorldKorea/RMK/assets/150018627/15f096d8-83ad-4b1e-8c0e-4168ba86d64a">

큰 글씨로 노트 달아둔거 번호 순서대로 보면 됨
